backend/ethereum/channel/funder.go

```diff
@@ -59,6 +62,12 @@ type Funder struct {
 var _ channel.Funder = (*Funder)(nil)
 
 // NewFunder creates a new ethereum funder.
+//
+// In addition to the `Fund` method required by the `Funder` interface, it also
+// providers two additional functions for convenience: RegisterAsset and
+// IsAssetRegistered.
```

Comment:
I think this comment is not necessary and hard to maintain. Not necessary because the doc will also list the functions. And hard to maintain because if we change the function, we need to change this comment which is easy to miss.

Reply:
I agree with on the difficulty in maintaining this comment. Maybe we should change this to 
```
In addition to `Fund` method required by the `Funder` interface, it provides additional methods for convenience.
```

To indicate there are other methods ? Or can it be removed completely ?

Reply:
If it is to be included, should these lines also be moved to the `Funder` type ?

Reply:
Updated in this [force-push](https://github.com/hyperledger-labs/go-perun/compare/c5255fbe5f5453796953eb1c9e559a9565a7c2ea..4a12ab6d8062a3296b14d92a697dc705aec0b7e5).